docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/index.md

```diff
@@ -0,0 +1,173 @@
+Simply put, what the Snowplow Inspector extension does is parse these HTTP requests into a more easily readable format.
+This allows anyone with the extension to more easily see what actions on a webpage with Snowplow tracking trigger events to be sent to a Snowplow pipeline. 
+
+In the example below two events were fired as HTTP post requests from a test webpage: a [**page_view**](/docs/understanding-your-pipeline/canonical-event/index.md#page-views) event and a ([Self Describing](/docs/understanding-your-pipeline/canonical-event/index.md#custom-unstructured-events)) [**link_click**](/docs/collecting-data/collecting-from-own-applications/javascript-trackers/javascript-tracker/javascript-tracker-v3/plugins/link-click-tracking/index.md) event. 
+
+![A screenshot of the Snowplow Inspector extension listing two events observed on the current page. A page_view event is selected, detailing the properties collected as a part of that event.](images/using-poplin-chrome-extension.png)
+
+Within the parsed out HTTP requests that comprise the event payload are a variety of [Snowplow canonical event](/docs/understanding-your-pipeline/canonical-event/index.md) fields such as unique `event_id`, timestamps, and user and session identifiers, as well as any custom [Event](/docs/understanding-your-pipeline/glossary-of-terms/index.md#self-describing-event) or [Entity](/docs/understanding-your-pipeline/glossary-of-terms/index.md#entity) fields.
+
+## The role of Snowplow Inspector in event QA
+
+The Snowplow Inspector extension is an invaluable tool for anyone who is pushing tracking live as it gives real time - easy to access - information on which events are being emitted and what the contents of these events are.
+This makes the tool a good first port of call when trying to answer questions such as “why is X event not appearing in my data warehouse” by showing whether or not the event is actually firing on a chosen webpage.  
+
+Additionally the extension can be configured to show whether or not an event has passed validation according to any event validation rules codified in the corresponding JSON Schema stored in an [Iglu](/docs/understanding-your-pipeline/glossary-of-terms/index.md#iglu) registry. 
+
+For events that failed validation in production historically that you are unable to replicate in your own browser, see our guides on [how to query failed events](/docs/managing-data-quality/failed-events/failed-events-in-athena-and-bigquery/index.md) from their respective destinations.
+These failed events have a [specific format](/docs/managing-data-quality/failed-events/understanding-failed-events/index.md) that includes an array of helpful, detailed error messages that explain the exact reasons why the event failed validation.
+These events can also be imported into the extension to view as if your browser had generated them itself.
+
+## Event validation with custom Iglu Registries
```

Comment:
I would remove "Iglu" from the headline here as it could be confusing (local repos and the DS API doesn't match well with the "Iglu Registry" description)